server/signedcontainer/internal/convert/convert.go

```diff
@@ -0,0 +1,262 @@
+package convert
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/tink-crypto/tink-go/v2/keyset"
+	tinkecdsa "github.com/tink-crypto/tink-go/v2/signature/ecdsa"
+	"github.com/tink-crypto/tink-go/v2/signature/rsassapkcs1"
+	"github.com/tink-crypto/tink-go/v2/signature/rsassapss"
+)
+
```

Comment:
Create an issue and add a TODO to migrate to the Tink API if/when they publish it.

Reply:
https://github.com/GoogleCloudPlatform/confidential-space/issues/10